src/App.tsx

```diff
@@ -1,26 +1,428 @@
+              />
+            </form>
+
+            <div data-cy="TodoLoader" className="modal overlay">
+              <div className="modal-background has-background-white-ter" />
+              <div className="loader" />
+            </div>
+          </div> */}
+
+              {/* This todo is in loadind state */}
+              {tempTodo && (
+                <div data-cy="Todo" className="todo">
+                  <label className="todo__status-label">
+                    <input
+                      data-cy="TodoStatus"
+                      type="checkbox"
+                      className="todo__status"
+                    />
+                  </label>
 
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+                  <span data-cy="TodoTitle" className="todo__title">
+                    {tempTodo.title}
+                  </span>
+
+                  <button
+                    type="button"
+                    className="todo__remove"
+                    data-cy="TodoDelete"
+                  >
+                    ×
+                  </button>
+
+                  {/* 'is-active' class puts this modal on top of the todo */}
+                  <div data-cy="TodoLoader" className="modal overlay is-active">
+                    <div
+                      className="modal-background
+                     has-background-white-ter"
+                    />
+                    <div className="loader" />
+                  </div>
+                </div>
+              )}
+            </section>
+
+            {/* Hide the footer if there are no todos */}
+
+            {todos?.length > 0 && (
+              <footer className="todoapp__footer" data-cy="Footer">
+                <span className="todo-count" data-cy="TodosCounter">
+                  {activeTodos.length} items left
+                </span>
+
+                {/* Active link should have the 'selected' class */}
+                <nav className="filter" data-cy="Filter">
+                  <a
+                    href="#/"
+                    className={classNames('filter__link', {
+                      selected: activeFilter === 'All',
+                    })}
+                    data-cy="FilterLinkAll"
+                    onClick={() => setActiveFilter('All')}
+                  >
+                    All
+                  </a>
+
+                  <a
+                    href="#/active"
+                    className={classNames('filter__link', {
+                      selected: activeFilter === 'Active',
+                    })}
+                    data-cy="FilterLinkActive"
+                    onClick={() => setActiveFilter('Active')}
+                  >
+                    Active
+                  </a>
+
+                  <a
+                    href="#/completed"
+                    className={classNames('filter__link', {
+                      selected: activeFilter === 'Completed',
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method